src/shim/runtime/v2/rune/constants/constants.go

```diff
@@ -0,0 +1,5 @@
+package constants
+
+const (
+	RuneOCIRuntime = "rune"
```

Comment:
we have talked whether we could just go with `runc`. how about this is made configurable in this rev? it would be easier to, e.g., run the unit tests.

Reply:
hi Mikko
We will use rune in the first release. And runc will be the next plan. So let us use the rune at present

Reply:
I think it needs to be at least configurable to get the unit tests in CI working without dependencies to rune.